mergify/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # CHANGELOG - Mergify
 
+## 0.1.1
```

Comment:
Small nit: I would consider this a breaking change, since you are removing the old `queue_checks_outcome` metrics, so this should be 1.0.0 instead of 0.1.1. However, since @sileht approved already I think this isn't necessary.

Reply:
Nice catch, thanks. I modified it. He approved when they weren't the `queue_checks_outcome` change yet.